onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -1030,7 +1031,7 @@ void TensorrtExecutionProvider::RemoveTensorRTGraphCycles(SubGraphCollection_t&
     }
 
     // Create adjacency list
-    int graph_size = node_to_index_map.size();
+    int graph_size = static_cast<int>(node_to_index_map.size());
```

Comment:
avoid casting from size_t to int to address these.

Reply:
Changed to use size_t.